### PR TITLE
Updated dependencies, uses react-dnd >= 7.1

### PR DESCRIPTION
**Pull request by dolezel**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { findDOMNode } from 'react-dom';
 import throttle from 'lodash.throttle';
 import raf from 'raf';
 import getDisplayName from 'react-display-name';
-import { Consumer as DragDropContextConsumer } from 'react-dnd/lib/DragDropContext';
+import { Consumer as DragDropContextConsumer } from 'react-dnd/lib/cjs/DragDropContext';
```
Comment (intizarahmad) [inline src/index.js:7]: We can also import by the direct react-dnd path
import { Consumer as DragDropContextConsumer } from 'react-dnd'; 

In this way both react-dnd versions (7.0.2 the older and 7.2.0 the latest) will work.

Comment (dolezel) [reply src/index.js:7]: Unfortunately Consumer is not exported from react-dnd

Comment (intizarahmad) [reply src/index.js:7]: @dolezel : Its exported from react-dnd, I tried to build it on my machine its working perfect. 
screenshot 2019-03-08 at 3 19 56 pm

Comment (dolezel) [reply src/index.js:7]: You are looking on compiled content of DragDropContext.tsx
if you check what is exported from react-dnd (`console.log(require('react-dnd'))`), it is this: 
```
DragDropContext: ƒ DragDropContext(backendFactory, backendContext, debugMode)
DragDropContextProvider: ƒ (_a)
DragLayer: ƒ DragLayer(collect, options)
DragSource: ƒ DragSource(type, spec, collect, options)
DropTarget: ƒ DropTarget(type, spec, collect, options)
__esModule: true
__proto__: Object
```

**Comment by kserjey** [comment]
They have published new version of [react-dnd](https://github.com/react-dnd/react-dnd/releases/tag/v7.3.1), maybe it worth to update then?

**Pull request closed without merging**